eng/common/scripts/Service-Level-Readme-Automation.ps1

```diff
@@ -105,6 +114,9 @@ function generate-markdown-table($readmeFolder, $readmeName, $packageInfo, $moni
   # Here is the table, the versioned value will
   foreach ($pkg in $packageInfo) {
     $repositoryLink = $RepositoryUri
```

Comment:
```suggestion
    $repositoryLink = "$RepositoryUri/$($pkg.Package)"
```

This this be the default implementation like @danieljurek suggested so that everything works as it used to and only java and android need to provide this custom function? Although from a quick search I cannot find where RepositoryUri is set so I'm not sure if this works or not. Where is RepositoryUri set?

Reply:
It is in Language-Setting.ps1

Reply:
https://github.com/sima-zhu/azure-sdk-for-js/blob/toc_service_update/eng/scripts/Language-Settings.ps1#L9